opencsp/common/lib/csp/FacetEnsemble.py

```diff
@@ -194,19 +183,16 @@ def draw(
             corners_moved = transform.apply(corners)
             view.draw_Vxyz(corners_moved, close=True, style=facet_ensemble_style.outline_style)
 
-    # debug function
     def set_facet_transform_list(self, transformations: list[TransformXYZ]):
         """
         Combines the `set_facet_positions` and `set_facet_canting` functions into a single action.
         """
-        warn("Untested function")
         for transformation, facet in zip(transformations, self.facets):
             facet._self_to_parent_transform = transformation
 
     def set_facet_positions(self, positions: Pxyz):
-        """
-        The function for setting the positions of the facets relative to one another.
-
+        """Sets the positions of the facets relative to the ensemble.
+        NOTE: Will remove previously set facet canting rotations
```

Comment:
if set_facet_positions() removes the canting, and set_facet_canting() removes the position, then how does both the position and canting get set at the same time?

Reply:
That is done by `set_facet_transform_list`
Not very intuitive. Maybe the focus of a future PR?

Reply:
That's a reasonable answer. Added a new issue #188